src/settings/mod.rs

```diff
@@ -205,6 +207,13 @@ pub struct Sendgrid {
     pub key: SendgridApiKey,
 }
 
+/// Settings for Sentry.
+#[derive(Debug, Default, Deserialize, Serialize)]
+pub struct Sentry {
+    /// The Sentry Data Source Name
+    pub dsn: SentryDsn,
```

Comment:
Should we have two separate DSN's, one for the service and another for the queues?

Reply:
@philbooth probably overkill